packages/web-components/src/components/va-telephone/va-telephone.tsx

```diff
@@ -124,15 +137,33 @@ export class VaTelephone {
     return labelPieces.join('. ');
   }
 
-  static createHref(contact: string, extension: number): string {
+  static createHref(contact: string, extension: number, sms: boolean): string {
     const isN11 = contact.length === 3;
     // extension format ";ext=" from RFC3966 https://tools.ietf.org/html/rfc3966#page-5
     // but it seems that using a comma to pause for 2 seconds might be a better
     // solution - see https://dsva.slack.com/archives/C8E985R32/p1589814301103200
-    const href = `tel:${isN11 ? contact : `+1${contact}`}`;
+    let href = `tel:+1${contact}`;
+    if (isN11) {
+      href = `tel:${contact}`;
+    } 
+    if (sms) {
+      href = `sms:${contact}`;
+    }
```

Comment:
I think this is a bit clearer and it has the benefit of keeping the SMS and N11 functionality separate, since I don't think you can text those kinds of numbers.

```suggestion
    let href = null;
    if (sms) {
      href = `sms:${contact}`;
    }
    else if (isN11) {
      href = `tel:${contact}`;
    }
    else {
      href = `tel:+1${contact}`;
    }
```

Reply:
I have updated the condition logic based on this suggestion.